### PR TITLE
insight_update

### PR DESCRIPTION
**Pull request by domdinicola**

### CODE REVIEW

File: src/unicef_vision/loaders.py [hunk 1]
```diff
@@ -32,7 +30,10 @@ def set_url(self, endpoint, detail):
         logger.info('About to get data from {}'.format(self.url))
 
     def set_headers(self, headers):
-        self.headers = {'Content-Type': 'application/json'}
+        self.headers = {
```
Comment (reinbach) [inline src/unicef_vision/loaders.py]: Any reason not to have this header in a single spot and reuse?

File: tests/test_loaders.py [hunk 2]
```diff
@@ -9,8 +9,7 @@
 from unicef_vision.loaders import FileDataLoader, ManualDataLoader, VISION_NO_DATA_MESSAGE, VisionDataLoader
 
 FAUX_VISION_URL = 'https://api.example.com/foo.svc/'
-FAUX_VISION_USER = 'jane_user'
-FAUX_VISION_PASSWORD = 'password123'
+FAUX_INSIGHT_SUB_KEY = 'insight_sub_key'
```
Comment (reinbach) [inline tests/test_loaders.py]: Why do we have this faux sub key? It appears to be the same as the normal sub key

File: src/unicef_vision/loaders.py (hunk 1 above)
Comment (domdinicola) [reply src/unicef_vision/loaders.py]: what do you mean?

Comment (reinbach) [reply src/unicef_vision/loaders.py]:
```
{
    'Content-Type': 'application/json',
    'Ocp - Apim - Subscription - Key': settings.INSIGHT_SUB_KEY
}
```
Is used on the following lines;
https://github.com/unicef/unicef-vision/blob/6b33958655fef7c24dc5dbc69c75aea48fe96760/src/unicef_vision/utils.py#L24
https://github.com/unicef/unicef-vision/blob/6b33958655fef7c24dc5dbc69c75aea48fe96760/tests/test_loaders.py#L26
https://github.com/unicef/unicef-vision/blob/6b33958655fef7c24dc5dbc69c75aea48fe96760/tests/test_loaders.py#L84
https://github.com/unicef/unicef-vision/blob/6b33958655fef7c24dc5dbc69c75aea48fe96760/tests/test_loaders.py#L130

**Comment by reinbach** [comment]
> remove me! :)

@domdinicola you control you notifications via the un/watch button at the top of the repo

**Pull request merged**